docs/configuration/env.md

```diff
@@ -0,0 +1,19 @@
+# Environment
+
+# Target configuration
```

Comment:
Add a table with all used properties will be more readable i think

Reply:
![image](https://user-images.githubusercontent.com/3458542/194104162-03e3548b-5b3f-4bae-bd4c-7af131b5e243.png)